### PR TITLE
fix: default.metrics source is not imported when metrics are disabled…

### PR DESCRIPTION
**Pull request by pdelewski**
… and traces are enabled

##### Description

Fill in your description here.

---

##### Checklist

- [ ] Changelog updated

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by sumo-drosiek — COMMENTED** [review]
Could you add unit tests for that?
They should be added here:
https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/tests/helm

You can take https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/tests/helm/remote_write_proxy as example

**Comment by pdelewski** [comment]
> Could you add unit tests for that? They should be added here: https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/tests/helm
> 
> You can take https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/tests/helm/remote_write_proxy as example

ok

**Comment by swiatekm-sumo** [comment]
@pdelewski you can rebase on main and tests should pass. We should also backport this to v2 imo.

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-- fix(openshift): fix remote write proxy - use unprivileged NGINX [#2510][#2510]
+- fix(openshift): fix remote write proxy - use unprivileged NGINX [#2510], [#2510]
```
Comment (swiatekm-sumo) [inline CHANGELOG.md:29]: Why this change?

**Review by swiatekm-sumo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pdelewski) [reply CHANGELOG.md:29]: Actually, I don't remember exactly the reason now, but I think that some check was failing
